Jenkinsfile

```diff
@@ -692,7 +696,7 @@ pipeline {
                     }
                     agent{ label rocmnode("gfx94X") }
                     steps{
-                        buildHipClangJobAndReboot(build_type: 'debug', make_targets: Smoke_targets)
+                        buildHipClangJobAndReboot(build_type: 'debug', make_targets: Smoke_targets, needs_reboot:false)
```

Comment:
[Question] @cderb Can you please clarify why reboot is disabled for MI300 here and at some other places?

Reply:
This is to avoid disabling the machine, as sometimes the MI300 node isn't reconnecting after reboot.